medusa/version_checker.py

```diff
@@ -896,6 +896,12 @@ def set_newest_text(self):
         # if we're up to date then don't set this
         app.NEWEST_VERSION_STRING = None
 
+        if self.runs_in_docker() and (not self._cur_commit_hash or self._num_commits_behind > 0):
+            log.debug(u'There is an update available, medusa is running in a docker container, so auto updating is disabled.')
+            app.NEWEST_VERSION_STRING = 'There is an update available: please pull the latest docker image, ' \
+                                        'and rebuild your container to update'
```

Comment:
Maybe we should link to something here?

Reply:
Step by step. First prio, is to not have installations break.
I'd rather have users come over here, to ask how to update in docker for now.

Reply:
We could link to wiki?

Reply:
Okay, this works: 
![image](https://user-images.githubusercontent.com/1331394/48402711-963da600-e72c-11e8-9d02-28362d43147c.png)